contracts/src/periphery/EigenDACertVerifierV2.sol

```diff
@@ -32,9 +32,6 @@ contract EigenDACertVerifierV2 is IEigenDACertVerifierV2 {
     /// @notice The EigenDASignatureVerifier contract address
     IEigenDASignatureVerifier public immutable eigenDASignatureVerifierV2;
 
-    /// @notice The EigenDARelayRegistry contract address
-    IEigenDARelayRegistry public immutable eigenDARelayRegistryV2;
-
     /// @notice The EigenDA middleware OperatorStateRetriever contract address
     OperatorStateRetriever public immutable operatorStateRetrieverV2;
```

Comment:
Btw unlreated to this PR but is there not a naming convention in solidity contracts for global variables (contract storage slots). I find it hard when reading a function to just see a bunch of variables, and I don't know which ones were passed as arguments vs which one are contract variables (and on top of that which of the contract variables are mutable vs immutable). Feels like a naming convention could help here?

Reply:
Eventually I'd like to transition to using unstructured storage only, not declaring any storage as public, etc., because I hate dealing with these kinds of questions haha. But for the codebase as it currently is, I think this naming makes the most sense for now. Seems like a separate issue to tackle.